Base/res/html/misc/welcome.html

```diff
@@ -151,6 +151,7 @@ <h2>JavaScript/Wasm</h2>
             <li><a href="location.html">window.location property</a></li>
             <li><a href="script-preparation-test.html">Test for the early return steps 6-8 of the "prepare a script" algorithm</a></li>
             <li><a href="async-js.html">Basic test for async functions and their integration with the LibWeb event loop</a></li>
+            <li><a href="worker_parent.html">Workers</a></li>
```

Comment:
This is being moved from the bottom to here in the second commit - please avoid that, it just adds noise to the diff.

Reply:
This was a previous suggestion to move the Workers page out of the wasm sub-section and into the main JS section.

Reply:
Sure, but currently it gets added to the bottom in one commit and then moved around in the second. We want to avoid that, so you'll need to squash code review changes into the appropriate commit :)